src/VisualizationTests/CameraTests.cs

```diff
@@ -48,11 +48,11 @@ public void Camera_GoodSaveData_LoadsCorrectly()
 
             var cam = ((HelixWatch3DViewModel)ViewModel.BackgroundPreviewViewModel).Camera;
 
-            //<Cameras>
+            //<Camera>
```

Comment:
When you deserialize the the element `Cameras` in XML - iterate it and just assign it to the new variable. The XML deserialization we currently have in Dynamo is very manual and you have control over how the deserialization occurs and how it is mapped to objects in memory.